apps/OpenSpace/ext/launcher/src/filesystemaccess.cpp

```diff
@@ -49,7 +49,8 @@ std::string FileSystemAccess::useQtFileSystemModelToTraverseDir(std::string dir)
 
 void FileSystemAccess::parseChildDirElements(QFileInfo fileInfo, std::string space,
                                          int level, std::vector<std::string>& dirNames,
-                                         std::vector<std::string>& output)
+                                         std::vector<std::string>& output,
+                                         bool userAsssets)
```

Comment:
Nitpicky;  seems like the `bool userAssets` could be on the same line as the `output` parameter.
And:  userAsssset has one s too many